compilers/compiler.js

```diff
@@ -31,4 +34,16 @@ exports.compile = function (load, opts, plugin) {
     source: output.code,
     sourceMap: sourceMap
   });
-};
\ No newline at end of file
+};
+
+exports.transformAMDDependencies = function (load) {
+  var options = {
+    plugins: [[require('babel-plugin-transform-amd-system-dependencies').default]]
+  };
+
+  var output = babel.transform(load.source, options);
+
+  return {
+    amdDeps: output.metadata.amdDeps
+  };
+}
```

Comment:
Sounds very good! This could be a real performance gain! The function name `transformAMDDependencies` already had the appearance not to belong in the compiler part.